src/components/DRB/DRBCard.tsx

```diff
@@ -0,0 +1,80 @@
+import {
+  Card,
+  CardContent,
+  Text,
+  Icon,
+  Link as DSLink,
+} from "@nypl/design-system-react-components"
+
+import type DRBResult from "../../models/DRBResult"
+import { getAuthorURL } from "../../utils/drbUtils"
+import type { Author, Agent } from "../../types/drbTypes"
+
+interface DRBCardProps {
+  drbResult: DRBResult
+}
+
+/**
+ * The DRBContainer fetches and displays DRBContainer search results
+ */
+const DRBCard = ({ drbResult }: DRBCardProps) => {
+  return (
+    <Card
+      backgroundColor="var(--nypl-colors-ui-bg-default)"
```

Comment:
The style prop shorthand values don't work?

Reply:
Fixed!